testsrc/by/javatr/model/BallTest.java

```diff
@@ -2,12 +2,18 @@
 
 import by.javatr.model.exception.InvalidColorException;
 import by.javatr.model.exception.InvalidWeightException;
+import by.javatr.model.validation.BallValidationUtils;
 import org.junit.Assert;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
 
 /**
  * @author Aleh Yemelyanchyk on 12/1/2019.
  */
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Do you use mocks in this test? If not just simple JUnitRunner should be good enough